barman/command_wrappers.py

```diff
@@ -1200,6 +1200,8 @@ def shell_quote(arg):
     # a backslash, and then start another string using a quote character.
 
     assert arg is not None
+    if arg == "|":
+        return arg
```

Comment:
I am probably missing something but is this change related to something else which changed in this PR?

Reply:
Well the change is actually not used. 
At first instead of using `tar --use-compress-program lz4 ...` I tried to use `lz4 -d file.tar.lz4 | tar -xf` 
I realized that `|` as argument was `"|"`  and it created an error at execution time.

I left it there, just in case we would need it at some point.
Do you think it should be removed?

Reply:
I think it should stay - I just felt incomplete for not knowing the story behind it :)